packages/squiggle-lang/__tests__/Distributions/KlDivergence_test.res

```diff
@@ -60,6 +60,48 @@ describe("kl divergence", () => {
   })
 })
 
+describe("kl divergence on discrete distributions", () => {
```

Comment:
Btw, I realize you've done this elsewhere, but I prefer naming styles that use the function names, instead of descriptions of the function names. With ruby, sometimes the describe block would just be, 
```
describe("#klDivergence"){
  context("with discrete distributions")....
} 
```